site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/widgets/Banner.ts

```diff
@@ -0,0 +1,15 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+
+export default class Banner extends WidgetBase<WidgetProperties> {
+	render() {
+		return v('div', {}, [
+			v('h1', {
+				styles: {
+					'text-align': 'center'
```

Comment:
This was added in the third tutorial, and I had assumed it was to show that it's possible to do inline styles, since other widgets use theming. If Mike wants me to, I can change it either just in the fifth tutorial, or go back and remove the inline style from all previous tutorials.